dbos/_dbos_config.py

```diff
@@ -52,6 +92,7 @@ class TelemetryConfig(TypedDict, total=False):
     OTLPExporter: Optional[OTLPExporterConfig]
 
 
+# FIXME: we should remove total=False and make database & runtimeConfig optional.
```

Comment:
Are FIXME's to be fixed before merging this PR or in a future PR?

Reply:
I'd like to make this change in a future PR. The main challenge with this fix is: it's not backward compatible..